src/assets/scripts/game/GameClockView.js

```diff
@@ -0,0 +1,96 @@
+import $ from 'jquery';
+import { digits_integer } from '../utilities/radioUtilities';
+import { SELECTORS } from '../constants/selectors';
+import { TIME } from '../constants/globalConstants';
+
+/**
+ * @class GameClockView
+ */
+export default class GameClockView {
+    /**
+     * @for GameClockView
+     * @constructor
+     */
+    constructor() {
+        this.$element = null;
+        this.startTime = 0;
+        this.time = 0;
+
+        return this._init();
+    }
+
+    /**
+     * @for GameClockView
+     * @method _init
+     * @private
+     */
+    _init() {
+        this.$element = $(SELECTORS.DOM_SELECTORS.CLOCK);
```

Comment:
now that we have `$element`, which is the `body` tag, we can leverage jquery `.find()` to start searching for our selector. 
```javascript
this.$element = $(SELECTORS.DOM_SELECTORS.CLOCK);
// becomes
this.$element = $element.find(SELECTORS.DOM_SELECTORS.CLOCK);
```
this give us a slight performance boost by starting the search for our selector at the body DOM node instead of starting from the root HTML node.  it's slight, but these slight boosts add up in the end.